tsne/bh_sne_src/tsne.cpp

```diff
@@ -758,41 +806,92 @@ void run(double* X, int N, int D, double* Y, double perplexity, double theta,
       if (iter == 0)
         fprintf(stderr, "Iteration %d: error is %f\n", iter + 1, C);
       else {
-        total_time += (float)(end - start) / CLOCKS_PER_SEC;
+        elapsed = (float)(end - start) / CLOCKS_PER_SEC;
```

Comment:
Again, to make order of operations more obvious,
```suggestion
        elapsed = static_cast<float>(end - start) / CLOCKS_PER_SEC;
```
I think there might even be a fixer in clang-tidy or something to do all of those automatically?